api/sys_capabilities.go

```diff
@@ -1,12 +1,14 @@
 package api
 
-import "fmt"
+import (
+	"fmt"
+)
 
-func (c *Sys) CapabilitiesSelf(path string) ([]string, error) {
+func (c *Sys) CapabilitiesSelf(path string) (map[string][]string, error) {
```

Comment:
This is an API break. If you want this functionality you'll need to make new methods.

Reply:
Thanks. That’s the bit I was most looking for feedback on. I’ll add new methods for this :)